src/OpenConext/ProfileBundle/Resources/config/routing.yml

```diff
@@ -1,6 +1,18 @@
-profile_homepage:
+profile_index:
     path:     /
-    defaults: { _controller: OpenConextProfileBundle:Default:index }
+    defaults: { _controller: OpenConextProfileBundle:Page:index }
+
+profile_my_profile:
+    path:     /my-profile
```

Comment:
Why two URLs and actions? I'd merge these into `/` and `myProfileAction`. Then you can use `app.request.attributes.get('_route')` to activate navigation links.